test/Microsoft.AspNetCore.SignalR.Tests/WebSocketsTransportTests.cs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using System.Threading;
```

Comment:
remove

Reply:
We need this for the Cts

Reply:
But looks like we should just be passing in `CancellationToken.None`  instead